src/navigation/courier/TaskListPage.js

```diff
@@ -29,108 +27,67 @@ const styles = StyleSheet.create({
   },
 });
 
-class TaskListPage extends Component {
-  completeSelectedTasks(selectedTasks) {
-    if (selectedTasks.length > 1) {
-      navigateToCompleteTask(
-        this.props.navigation,
-        this.props.route,
-        null,
-        selectedTasks,
-        true,
-      );
-    } else if (selectedTasks.length === 1) {
-      navigateToCompleteTask(
-        this.props.navigation,
-        this.props.route,
-        selectedTasks[0],
-        [],
-        true,
-      );
-    }
-  }
+const allowToSelect = task => {
+  return task.status !== 'DONE';
+};
 
-  allowToSelect(task) {
-    return task.status !== 'DONE';
+export default function TaskListPage({ navigation, route }) {
+  const selectedDate = useSelector(selectTaskSelectedDate);
+  const tasks = useSelector(selectFilteredTasks);
+  const tasksWithColor = useSelector(selectTasksWithColor);
+
+  const containerStyle = [styles.container];
+  if (tasks.length === 0) {
+    containerStyle.push(styles.containerEmpty);
   }
 
-  render() {
-    const { tasks, tasksWithColor, selectedDate } = this.props;
+  const { isFetching, refetch } = useGetMyTasksQuery(selectedDate, {
+    refetchOnFocus: true,
```

Comment:
@vladimir-8 we should enable `refetchOnReconnect` here as well no ?

Reply:
We can do it, why not